### ISSUE TITLE
Import of StreamingJob fails with incorrect reason when import id is incorrect

### ISSUE THREAD

**Issue by WillooWisp** [pulumi/pulumi-azure-nextgen#200]
error: Preview failed: autorest/azure: Service returned an error. Status=400 Code="InvalidApiVersionParameter" Message="The api-version '2016-03-01' is invalid. The supported versions are '2020-10-01,2020-09-01,2020-08-01,2020-07-01,2020-06-01,2020-05-01,2020-01-01,2019-11-01,2019-10-01,2019-09-01,2019-08-01,2019-07-01,2019-06-01,2019-05-10,2019-05-01,2019-03-01,2018-11-01,2018-09-01,2018-08-01,2018-07-01,2018-06-01,2018-05-01,2018-02-01,2018-01-01,2017-12-01,2017-08-01,2017-06-01,2017-05-10,2017-05-01,2017-03-01,2016-09-01,2016-07-01,2016-06-01,2016-02-01,2015-11-01,2015-01-01,2014-04-01-preview,2014-04-01,2014-01-01,2013-03-01,2014-02-26,2014-04'."
error: preview failed

**Comment by mikhailshilkov** [comment]
This is tracked in https://github.com/pulumi/pulumi-azure-nextgen/issues/191

For now, change the import urn from `latest` to `v20201001`

**Comment by mikhailshilkov** [comment]
Oh, sorry, I've misread... ignore the previous comment

**Comment by mikhailshilkov** [comment]
We query the URL with the ID that you provided and return the error back if we get one. We could probably add validation for the ID format that you provide.

Side note - the correct format of IDs is available in the docs at the very bottom of resource pages, e.g. https://www.pulumi.com/docs/reference/pkg/azure-nextgen/streamanalytics/streamingjob/#import

**Comment by WillooWisp** [comment]
Yes, all good, it was my mistake, but took me some time to understand what the problem was since the error message pointed me in the wrong direction.
